MAINTAINERS

```diff
@@ -1,2 +1,3 @@
+Ben Darnell <ben@bendarnell.com>
```

Comment:
Yes, either note that it's just for the raft package or if you can't change the format of the file move me to the end of the list instead of alphabetical order. Also I'd prefer to use the address ben@cockroachlabs.com here.